FreeRTOS/FreeRTOS-Cellular-Interface
# Merge update version to v1.4.1-patch branch.

Merge update version PR

Description
-----------

Merge #187 to branch v1.4.1-patch for v1.4.1 release.

Test Steps
-----------

Run doxygen locally.

Checklist:
----------



Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
